src/esp/bindings/bindings.cpp

```diff
@@ -69,6 +69,7 @@ PYBIND11_MODULE(habitat_sim_bindings, m) {
   esp::physics::initPhysicsBindings(m);
   esp::scene::initSceneBindings(m);
   esp::sensor::initSensorBindings(m);
+  esp::sim::initSimBindings(m);
```

Comment:
I believe this needs to be moved before init sensor bindings for doc generation to cross-link correctly.

Reply:
Actually, it looks like we have a lot of things that aren't cross linking correctly anymore :/

Reply:
How does the cross-linking work? How do I verify it?

Reply:
Talked to @erikwijmans offline. Will fix cross-linking in a separate change.